boards/nordic/nrf54l09pdk/nrf54l09pdk_nrf54l09_cpuflpr.dts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf54l09_enga_cpuflpr.dtsi>
+#include "nrf54l09pdk_nrf54l09-common.dtsi"
+
+/ {
+	model = "Nordic nRF54L09 PDK nRF54L09 FLPR MCU";
+	compatible = "nordic,nrf54l09pdk_nrf54l09-cpuflpr";
+
+	chosen {
+		zephyr,console = &uart30;
+		zephyr,shell-uart = &uart30;
+		zephyr,code-partition = &cpuflpr_code_partition;
+		zephyr,flash = &cpuflpr_rram;
+		zephyr,sram = &cpuflpr_sram;
+	};
+};
+
+&cpuflpr_sram {
+	status = "okay";
+};
+
+&cpuflpr_rram {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		cpuflpr_code_partition: partition@0 {
+			label = "image-0";
+			reg = <0x0 DT_SIZE_K(60)>;
```

Comment:
What's the point in having 60 kB here when only 48 kB of this partition can fit into SRAM?

Reply:
It's about RRAM, not SRAM: https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/common/nordic/nrf54l09.dtsi#L598

Reply:
I know, but in order to execute the code, the partition is copied to SRAM and it is in fact shrunk in the snippet: https://github.com/zephyrproject-rtos/zephyr/blob/75937c899817727d2b52941ae8866bfc9cb73a30/snippets/nordic-flpr/soc/nrf54l09_cpuapp.overlay#L11-L14
That's why I'm wondering if that 60 kB won't be actually confusing.